### ISSUE TITLE
Chaotic loading of a new comment

### ISSUE THREAD

**Issue by markoa** [operately/operately#897]
tl;dr why don't we just append?

Attaching two examples that show different glitches that I've seen:

https://github.com/user-attachments/assets/fe96db2b-5102-4ee0-b37c-12de84f10a1d

look at the different states the UI goes through:
<img width="1637" alt="Screenshot 2024-08-12 at 14 53 45" src="https://github.com/user-attachments/assets/e6ff7d12-97b2-4e4f-a480-894f34af98b6">
<img width="1637" alt="Screenshot 2024-08-12 at 14 53 50" src="https://github.com/user-attachments/assets/acd047e9-c67a-4681-8086-5f74d4cd4c45">
<img width="1637" alt="Screenshot 2024-08-12 at 14 53 57" src="https://github.com/user-attachments/assets/01346d85-e1ee-4d31-a36c-10fc266907a2">
<img width="1637" alt="Screenshot 2024-08-12 at 14 54 04" src="https://github.com/user-attachments/assets/0908b051-6c13-4da5-a567-3cfe22637b39">
<img width="1637" alt="Screenshot 2024-08-12 at 14 54 09" src="https://github.com/user-attachments/assets/f1f19ead-d7a5-4ba3-8c64-9e5bc7bd1440">

here's a different one:

https://github.com/user-attachments/assets/d852b9fd-95a3-493c-9cec-eec8c8ddc9ca

here the content in the comment box briefly remains on screen even after the comment section has been reloaded:

<img width="1637" alt="Screenshot 2024-08-12 at 14 58 20" src="https://github.com/user-attachments/assets/0f63f652-a595-4562-b217-43b685039bd3">
<img width="1637" alt="Screenshot 2024-08-12 at 14 58 24" src="https://github.com/user-attachments/assets/3fec908a-b88a-44d3-8940-4541f173f42f">
<img width="1637" alt="Screenshot 2024-08-12 at 14 58 28" src="https://github.com/user-attachments/assets/28295c6d-066c-4870-bcac-17f169c4e0e3">
<img width="1637" alt="Screenshot 2024-08-12 at 14 58 33" src="https://github.com/user-attachments/assets/e287f10d-302b-4616-999a-d5c8639fe966">
